styles/styles.css

```diff
@@ -450,15 +451,33 @@ main img {
 
 /* sections */
 
-main .section.padded > .section-container {
+main .section > .section-container {
   max-width: 81.25rem;
   width: 100%;
-  padding-left: 1rem;
-  padding-right: 1rem;
   margin-right: auto;
   margin-left: auto;
 }
 
+/* section - block-padded */
```

Comment:
Inline with previous comment, should the be block level styles?

Reply:
> Inline with previous comment, should the be block level styles?

done.